templates/etc/kubernetes/kubelet.erb

```diff
@@ -218,6 +218,15 @@ KUBELET_ARGS="<% -%>
 <% if @low_diskspace_threshold_mb -%>
  --low-diskspace-threshold-mb=<%= scope['kubernetes::node::kubelet::low_diskspace_threshold_mb'] -%>
 <% end -%>
+<% if @log_dir -%>
```

Comment:
If you already have this variable in the scope (`@log_dir`) you don't need to use `scope['kubernetes::node::kubelet::log_dir']` here, just use `@log_dir`.